Source/View+SnapKit.swift

```diff
@@ -178,6 +178,58 @@ public extension View {
             objc_setAssociatedObject(self, &installedLayoutConstraintsKey, newValue, objc_AssociationPolicy.OBJC_ASSOCIATION_RETAIN_NONATOMIC)
         }
     }
+    
+    internal var snp_constraints: [Constraint] {
```

Comment:
We shouldn't need this property on the view at all. There is a snp_layoutConstraints which you can map back into the originally Constraints. Was that not suitable?